028-HubAndSpoke/Student/02-AzFW.md

```diff
@@ -0,0 +1,33 @@
+# Challenge 2: Firewalling traffic

```

Comment:
Capitalize Traffic

Reply:
And same for all the other challenge titles when they're in section headers.

Reply:
Done